From 709d925de5b8f94e6669f3146a46875376deecdf Mon Sep 17 00:00:00 2001
From: Khai Wen Ng <khai.wen.ng@intel.com>
Date: Fri, 12 Sep 2025 03:31:10 +0800
Subject: [PATCH 54/61] media: i2c: maxim-serdes: max9296a: move use_atr to ops

use_atr should be in max_des_ops, instead of chip_info

Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/i2c/maxim-serdes/max9296a.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/maxim-serdes/max9296a.c b/drivers/media/i2c/maxim-serdes/max9296a.c
index 2efb70e89acb4..cfbf11adbb1ec 100644
--- a/drivers/media/i2c/maxim-serdes/max9296a.c
+++ b/drivers/media/i2c/maxim-serdes/max9296a.c
@@ -1169,12 +1169,12 @@ static const struct max_des_ops max9296a_ops = {
 	.num_pipes = 4,
 	.num_phys = 2,
 	.num_links = 2,
+	.use_atr = true,
 };
 
 static const struct max9296a_chip_info max9296a_info = {
 	.ops = &max9296a_ops,
 	.max_register = 0x1f00,
-	.use_atr = true,
 	.phy0_lanes_0_1_on_second_phy = true,
 	.pipe_hw_ids = { 0, 1, 2, 3 },
 	.phy_hw_ids = { 1, 2 },
-- 
2.50.1

